src/image/rotate_worker.cpp

```diff
@@ -16,11 +16,12 @@ void RotateWorker::Execute () {
     const float nangle = cimg::mod(_degs, 360.0f);
     if (cimg::mod(nangle, 90.0f) != 0) {
         CImg<unsigned char> * res;
+        int channels = _cimg->spectrum();
```

Comment:
No worries.
I like the code to be consistent, so if we leave it like this, we have to make similar changes in other places [in](https://github.com/kant2002/lwip/blob/master/src/image/image.cpp#L34) [the](https://github.com/kant2002/lwip/blob/master/src/image/pad_worker.cpp#L27) [code](https://github.com/kant2002/lwip/blob/master/src/image/crop_worker.cpp#L17).
I agree that this code gives us more flexibility regarding the number of channels, but we need to design those changes properly, as they affect other parts of the library.
Let's revert for now, and later decide how we properly bring this back across the library.
Thanks :)